ietf/api/views_rpc.py

```diff
@@ -83,16 +97,17 @@ def rpc_draft(request, doc_id):
             "stream": d.stream.slug,
             "title": d.title,
             "pages": d.pages,
+        "source_format": _document_source_format(d),
             "authors": [
                 {
                     "id": p.pk,
                     "plain_name": p.person.plain_name(),
                 }
                 for p in d.documentauthor_set.all()
             ],
-        }
-    )
-
+        "shepherd": d.shepherd.formatted_ascii_email() if d.shepherd else "",
```

Comment:
See my comment on the companion RPC PR - I'd rather not leak this information into the RPC app unless we really need to. The field will go through a big model change soon.

Reply:
The Document Shepherd email address is included in the CC of the AUTH48 messages

Reply:
We could fetch that directly from the datatracker when generating the message.

Reply:
Leaving the shepherd email address in the API so it can be shown, but no longer storing it on the RPC side at all.